app/src/main/java/org/coepi/android/ui/thanks/ThanksViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package org.coepi.android.ui.thanks
+
+import androidx.lifecycle.ViewModel
+import org.coepi.android.NavGraphRootDirections.Companion.actionGlobalHomeFragment
+import org.coepi.android.ui.alerts.AlertsFragmentDirections.Companion.actionGlobalAlerts
+import org.coepi.android.ui.navigation.NavigationCommand.ToDestination
+import org.coepi.android.ui.navigation.RootNavigation
+import org.coepi.android.ui.symptoms.SymptomsFragmentDirections.Companion.actionGlobalSymptomsFragment
+
+class ThanksViewModel(
+    private val navigation: RootNavigation
+) : ViewModel() {
+
+    fun onCheckInClick() {
+        navigation.navigate(ToDestination(actionGlobalSymptomsFragment()))
+    }
+
+    fun onSeeAlertsClick() {
+        navigation.navigate(ToDestination(actionGlobalAlerts()))
```

Comment:
Is the navigation being handled correctly? This seems like it lets the thanks fragment and the form in the back stack, which is probably not wanted.

Reply:
Adding `app:popUpTo="@id/tabs_container"` to each of those actions in nav_graph_root.xml fixes that behavior, so hitting the back button goes back to the home screen and never back to the thanks screen. Let me know if there's a better way of doing this.

Reply:
Hmm, we need to remove the remaining tabs setup from the navigation. Let's ignore this for now. I'll do it later and fix this as part of it.